Assets/Develop/Scripts/Familiar/FamiliarController.cs

```diff
@@ -59,25 +85,43 @@ public class FamiliarController : MonoBehaviour {
 
         }
 
+        if(player.transform.position.x > this.transform.position.x) {
+            skeletonMecanim.skeleton.ScaleX = 1 * controller.FlipGravityScale;
+        } else if (player.transform.position.x < this.transform.position.x) {
+            skeletonMecanim.skeleton.ScaleX = -1 * controller.FlipGravityScale;
+        }
```

Comment:
Should extract the common code, for example like this:
```suggestion
        float scaleDirection;
        if(player.transform.position.x > this.transform.position.x) {
            scaleDirection = 1;
        } else if (player.transform.position.x < this.transform.position.x) {
            scaleDirection = -1;
        }
        skeletonMecanim.skeleton.ScaleX = scaleDirection * controller.FlipGravityScale;
```
or like this:
```suggestion
        float scaleDirection = Mathf.Sign(player.transform.position.x - this.transform.position.x);
        skeletonMecanim.skeleton.ScaleX = scaleDirection * controller.FlipGravityScale;
```

Reply:
Suggestion 2 is a more elegant solution, and it also fixes the edge case mentioned above